content/40.inequality.md

```diff
@@ -182,31 +254,32 @@ This approach would carry its own ethical concerns, including the fact that many
 As a result, the standard practice of requiring enrollment in a clinical trial in order to receive experimental treatment may also reinforce patterns established by systemic racism.
-<!--Risk of comorbid health conditions associated with more severe outcomes may be influenced by long-term damage caused by chronic stress related to traumatic social experiences [@doi:10.1007/s12170-013-0338-5], perhaps mediated by cardiovascular risk factors [@doi:10.1101/2020.05.10.20097253], although the effects of chronic stress have not yet been researched in the specific case of COVID-19 disparities.-->
+<!-- To Do: move down: Risk of comorbid health conditions associated with more severe outcomes may be influenced by long-term damage caused by chronic stress related to traumatic social experiences [@doi:10.1007/s12170-013-0338-5], perhaps mediated by cardiovascular risk factors [@doi:10.1101/2020.05.10.20097253], although the effects of chronic stress have not yet been researched in the specific case of COVID-19 disparities.-->
 
 #### Access to and Representation in Clinical Trials
 
 Experimental treatments are often made available to patients primarily or even exclusively through clinical trials.
-The advantage of this approach is that clinical trials are designed to collect rigorous data about the effects of a treatment on patients.
+The advantage of this approach is that clinical trials are designed to collect rigorous data about the efficacy of a treatment as well as its safety.
 The disadvantage is that access to clinical trials is not equal among all people who suffer from a disease.
 Two important considerations that can impact an individual's access to clinical trials are geography and social perceptions of clinical trials.
 For the first, the geographic distribution of trial recruitment efforts are typically bounded and can vary widely among difference locations, and for the second, the social context of medical interactions can impact strategies for and the success of outreach to different communities.
 Differential access to clinical trials raises concerns because it introduces biases that can influence scientific and medical research on therapeutics and prophylactics broadly.
 Concerns about bias in clinical trials need to address both trial recruitment and operation.
-In the present crisis, such biases are particularly salient because COVID-19 is a disease of global concern.
+In the present crisis, such biases are particularly salient because COVID-19 is a disease of global concern, and COVID-19 studies that have included worldwide populations, such as the World Health Organization's (WHO's) Solidarity trial, do not always reach the same conclusions as those conducted exclusively within western, industrialized nations (e.g., remdesivir and tocilizumab <!--To Do: cite therapeutics-->).
```

Comment:
```suggestion
In the present crisis, such biases are particularly salient because COVID-19 is a disease of global concern, and COVID-19 studies that have included worldwide populations, such as the World Health Organization's (WHO's) Solidarity trial, do not always reach the same conclusions as those conducted exclusively within western, industrialized nations (e.g., remdesivir and tocilizumab [@individual-pharmaceuticals]).
```